source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -545,27 +665,49 @@ void InjaTransformer::transform(Http::RequestOrResponseHeaderMap &header_map,
     }
   }
   // get the extractions
-  std::unordered_map<std::string, absl::string_view> extractions;
+  std::unordered_map<std::string, std::string> extractions;
```

Comment:
Do we have any sense as to how much extra mem this will eat up? 
This should scale with number of extractions denoted right? So probably pretty low?

Reply:
Yeah, this is an additional string per extraction, though to be fair that string could potentially be quite large if the input is a large request body or something like that.

Reply:
`per_connection_buffer_limit_bytes` defaults to ~3.2kb (https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto), so I think it's reasonable to assume the upper limit for the size of this new string is around 10kb, with some caveats:
 - We should probably recommend that users don't use request/response bodies as inputs to extractors, especially if there is a chance that these bodies could be large
 - The size of these strings could grow larger if you made a serious mistake in configuration (i.e. write a regex that matches every character, then replace every match with a N-character string)
 - With the highlighted change, we are now converting the `absl::string_view` output of non-destructive extractions (i.e. the old extractor behavior) to `std::string` -- I'm currently looking into whether we can avoid this